### PR TITLE
ModelViewer move UiHelper to constructor 

### PR DESCRIPTION
**Pull request by srMarlins**
Moving `UiHelper` to the constructor which allows for more composable control of the underlying views & surface.

More specifically, I needed to set the opacity of the underlying views within `ModelViewer`.

### CODE REVIEW

File: android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt [hunk 1]
```diff
@@ -82,9 +85,9 @@ class ModelViewer(val engine: Engine) : android.view.View.OnTouchListener {
     val view: View
     val camera: Camera
     val renderer: Renderer
-    @Entity val light: Int
+    @Entity
```
Comment (romainguy) [inline android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt:88]: Please revert this change

**Review by romainguy — CHANGES_REQUESTED** [review]

File: android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt (hunk 1 above)
Comment (srMarlins) [reply android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt:88]: my bad, should I revert all the format changes?

Comment (romainguy) [reply android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt]: No just this one is fine, thanks!

**Review by romainguy — APPROVED** [review]

**Comment by srMarlins** [comment]
Android studio auto imported `import android.animation.Animator` without me realizing so had to delete that
